### PR TITLE
fix(pipes): add module exports for existing pipes

### PR DESCRIPTION
**Pull request by st3fun1**
# Description

This PR removes unused **pipe.module** file and fixes the imports of the **pipe** modules.

## Requirements

Storybook link: (once netlify has deployed link provide a link to the component)

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [x] I have added any new public feature modules to public-api.ts

### CODE REVIEW

File: projects/canopy/src/lib/pipes/camel-case/camel-case.notes.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ This pipe allows for a string to be transformed into camel case.
 Import the pipe in your module:
 
 ~~~
+import { LgCamelCasePipeModule } from '@legal-and-general/canopy';
```
Comment (elenagarrone) [inline projects/canopy/src/lib/pipes/camel-case/camel-case.notes.ts:11]: We never write the import in the documentation as it's usually automatically added by the IDE in your codebase so for consistency I would keep the notes as they were before.

File: projects/canopy/src/lib/pipes/index.ts [hunk 2]
```diff
@@ -1,2 +1,4 @@
+export * from './camel-case/camel-case.module';
```
Comment (elenagarrone) [inline projects/canopy/src/lib/pipes/index.ts:1]: 👏

**Review by elenagarrone — CHANGES_REQUESTED** [review]
Would you mind also squashing and updating the commit message to be:
```
fix(pipes): add module exports for existing pipes
```

File: projects/canopy/src/lib/pipes/camel-case/camel-case.notes.ts (hunk 1 above)
Comment (st3fun1) [reply projects/canopy/src/lib/pipes/camel-case/camel-case.notes.ts:11]: it wasn't obvious how to import the pipe, that's how I found the bug

**Review by elenagarrone — APPROVED** [review]

**Review by paul-request — APPROVED** [review]

**Pull request merged**